molecule/default/tests/test_default.py

```diff
@@ -16,5 +16,8 @@
     "f", ["/usr/bin/python", "/usr/bin/python2", "/usr/bin/python2.7"]
 )
 def test_python(host, f):
-    """Ensure that python2-specific files no longer exist."""
-    assert not host.file(f).exists
+    """Ensure that python2-specific files no longer exist, except on Debian 9."""
+    if host.system_info.distribution == "debian" and host.system_info.release == "9.12":
```

Comment:
Debian 9.13 was released on the 18th: https://www.debian.org/News/2020/20200718

Reply:
Support for 9.13 added in commit e2adf63.